src/Radar.js

```diff
@@ -178,7 +193,7 @@ export default class Radar extends Viz {
         .map(l => `L ${l.x} ${l.y}`)
         .join(" ")} L ${q[0].x} ${q[0].y}`;
 
-      return {id: h.key, d};
+      return {[id]: h.key, d};
```

Comment:
We need to make sure to provide the user data with the data points given to `Path`, as the user may have shapeConfigs setup that reference the internal data. If you change this line to the following, you'll also be able to get rid of the `id` variable logic, which is unreliable because the user could pass an accessor as a groupBy so there might not be a specific key it maps to:

```js
return {id: h.key, d, __d3plusShape__: true, data: h.values};
```

Reply:
@cnavarreteliz can you send me your sample test code/data? It worked for me as long as I provided `__d3plusShape__: true`

Reply:
@cnavarreteliz try this line (import `merge` from `d3-array`):

```js
return {id: h.key, d, __d3plus__: true, data: assign(...merge(h.values.map(d => d.values)))};
```

The `__d3plus__` tells all of the internal logic (like shape fills) to look for a `data` key that contains the user data. The issue here is that that data value needs to be a single data point, not multiple.

Reply:
This also allow users to override (the internals pass only the user's data object to the accessor, not out internal object):

```js
.shapeConfig({
      fill: d => d.metric === "Riverside" ? "red" : "blue"
    })
```